.github/workflows/e2e.yml

```diff
@@ -19,7 +19,7 @@
             }
 
     run-e2e-tests:
-      if: github.event.repository.name != 'terraform-azurerm-avm-template' && github.event.pull_request.head.repo.fork == false
+      if: github.event.pull_request.head.repo.fork == false
```

Comment:
@lonegunmanb could you please have a look at this comment by @zioproto 

https://github.com/Azure/terraform-azurerm-avm-ptn-aks-production/pull/141#discussion_r1848169635